perceval/utils.py

```diff
@@ -49,6 +50,18 @@
 DEFAULT_LAST_DATETIME = datetime.datetime(2100, 1, 1, 0, 0, 0,
                                           tzinfo=dateutil.tz.tzutc())
 
+GITHUB_API_URL = 'https://api.github.com'
+INSTALLATION_URL = GITHUB_API_URL + '/app/installations'
+ACCESS_TOKEN_URL = INSTALLATION_URL + '/{}/access_tokens'
+AUTH_URL = GITHUB_API_URL + '/installation/repositories'
```

Comment:
This isn't valid because the apps can be installed in GitHub enterprise instances too.